spec/javascripts/saveSpec.js

```diff
@@ -255,4 +254,21 @@ describe('Saving a resource instance', function() {
     });
 
   });
+
+  describe('payload for save requests', function() {
+    var resource;
+    beforeEach(function() {
+      resource = Model.create({ name: 'foo', id: 12, subject: 'hello world!' });
+    });
+
+    it('saves requested fields when specified', function() {
+      resource.save({fields: ['name', 'id']});
```

Comment:
Would we open source `EditableResource` to include it here? Can you point me to an example situation where we want to save a model but we don't know what changed?